packages/langium/src/validation/document-validator.ts

```diff
@@ -232,13 +233,13 @@ export function getDiagnosticRange<N extends AstNode>(info: DiagnosticInfo<N, st
 export function toDiagnosticSeverity(severity: 'error' | 'warning' | 'info' | 'hint'): DiagnosticSeverity {
     switch (severity) {
         case 'error':
-            return DiagnosticSeverity.Error;
+            return 1; // according to vscode-languageserver-types/lib/esm/main.js#DiagnosticSeverity.Information
```

Comment:
I now replaced the constants like `DiagnosticSeverity.Information` from `vscode-languageserver-types` by their values, and left an info on their original definition.